opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -2866,11 +2867,17 @@ namespace Opm {
             for (auto gi = 0*nu; gi < nu; ++gi) {
                 const auto& gname = up[start + gi];
                 const auto& group = sched.getGroup(gname, reportStepIdx);
+                const auto  grup_type = group.getGroupType();
```

Comment:
This variable is not referenced later.  Is that intentional?  If so, we should just remove it.